src/OpenTelemetry.Extensions.Enrichment/OpenTelemetry.Extensions.Enrichment.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(NetFrameworkMinimumSupportedVersion);$(NetStandardMinimumSupportedVersion)</TargetFrameworks>
+    <Description>OpenTelemetry .NET SDK telemetry enrichment.</Description>
+    <NoWarn>$(NoWarn),CS1591</NoWarn>
+    <MinVerTagPrefix>Extensions.Enrichment-</MinVerTagPrefix>
+    <Nullable>enable</Nullable>
+    <EnableAnalysis>true</EnableAnalysis>
```

Comment:
nit: Do `<AnalysisLevel>latest-all</AnalysisLevel>` instead of `<EnableAnalysis>true</EnableAnalysis>`

Reply:
Done